src/views/settings/SettingsNotifications.tsx

```diff
@@ -44,40 +86,51 @@ const SettingsNotifications: Screen<"SettingsNotifications"> = () => {
       return;
     }
 
-    if (await requestNotificationPermission()) return;
-
-    await mutateProperty("personalization", { notifications: { ...notifications, enabled: newValue } });
     setEnabled(newValue);
   };
 
   // Schoolary notifications
   const notificationSchoolary = [
-    // {
-    //   icon: <NativeIcon icon={<CalendarCheck />} color={colors.primary} />,
-    //   title: "Modification de cours",
-    //   message: "Cours de mathématiques annulé dans 10 minutes",
-    //   personalizationValue: "timeTable",
-    // },
-    // {
-    //   icon: <NativeIcon icon={<BookCheck />} color={colors.primary} />,
-    //   title: "Travail à faire pour demain",
-    //   message: "N’oublie pas de terminer ton devoir de français pour demain",
-    //   personalizationValue: "homework",
-    // },
-    // {
-    //   icon: <NativeIcon icon={<TrendingUp />} color={colors.primary} />,
-    //   title: "Nouvelle note",
-    //   message: "Nouvelle note disponible : 18/20 en histoire",
-    //   personalizationValue: "grades",
-    // },
+    {
+      icon: <NativeIcon icon={<CalendarCheck />} color={colors.primary} />,
+      title: "Emploi du temps du jour modifié",
+      message: "Le cours de Musique (10:00-11:00) a été annulé",
+      personalizationValue: "timetable",
+    },
+    {
+      icon: <NativeIcon icon={<BookCheck />} color={colors.primary} />,
+      title: "Nouveau devoir",
+      message: "Un nouveau devoir en Mathématiques a été publié",
+      personalizationValue: "homeworks",
+    },
+    {
+      icon: <NativeIcon icon={<TrendingUp />} color={colors.primary} />,
+      title: "Nouvelle note",
+      message: "Une nouvelle note en Anglais a été publiée",
+      personalizationValue: "grades",
+    },
     {
       icon: <NativeIcon icon={<Newspaper />} color={colors.primary} />,
       title: "Nouvelle actualité",
-      message: "Nouvelle actualité : \"Les élèves de 3ème partent en voyage scolaire\"",
+      message: "Chers élèves, chers collègues, Dans le cadre du prix \"Non au harcèlement\", 9 affiches ont été réa...",
       personalizationValue: "news",
     },
+    {
+      icon: <NativeIcon icon={<NotepadText />} color={colors.primary} />,
+      title: "Nouvelle événement sur la Vie Scolaire",
```

Comment:
si possible de corriger ca 
Nouvel événement sur la Vie Scolaire.

Reply:
```suggestion
      title: "Nouvel événement sur la Vie Scolaire.",
```

Reply:
```suggestion
      title: "Nouvel événement sur la Vie Scolaire",
```

Reply:
et peut etre remplacer sur par dans , mais je suis pas sur: 
```suggestion
      title: "Nouvel événement dans Vie Scolaire",
```